STYLE_GUIDE.md

```diff
@@ -0,0 +1,158 @@
+
+| File                 | Description                                | Standard                                         |
+|:---------------------|:-------------------------------------------|:-------------------------------------------------|
+| `README.md`          | Project overview, installation, and usage. | Markdown                                         |
+| `LICENSE`            | Legal terms of use.                        | **Apache License 2.0**                           |
+| `CONTRIBUTING.md`    | Guidelines for contributors.               | Link to org-wide guide or repo-specific details. |
+| `CODE_OF_CONDUCT.md` | Community standards.                       | Standard Contributor Covenant                    |
+| `SECURITY.md`        | Security policy.                           | Vulnerability reporting instructions             |
+| `.github/`           | GitHub configuration.                      | Issue templates, PR templates, workflows.        |
+
+> Content of all these files can link to org-wide files, and eventually incremented with repository specific content.
+
+---
+
+## 3. Contribution Workflow
+
+* **Branching Strategy:**
+    * **Main Branch:** `main` is the stable production branch.
+    * **Feature Branches:** Create branches from `main` for all changes.
+* **Pull Requests (PRs):**
+    * **Atomic Changes:** PRs should be small enough to be reviewed in one sitting. Large, sprawling PRs may be requested to be split.
+    * **Review Requirement:** All PRs require review from at least two Maintainers.
+    * **CI/CD Gates:**
+      * **Standard:** All PRs must generally pass automated checks (linting, testing, build) before merging.
+      * **Exceptions:** We recognize that checks may occasionally fail due to external issues outside our control or transient flakes that pass locally. In these rare instances, **maintainers can discuss and agree on exceptions** to merge specific PRs despite a red status.
+    * **Pull Request Title Format**
+      * **Format:** `<type>: <description>`
+      * **Example:** `feat: implement oscal validation logic`
+
+* **Commit Messages:**
+    * Follow the **Conventional Commits** [specification](https://www.conventionalcommits.org/).
+
+---
+
+## 4. Centrally manages configuration files, templates, and standardized CI/CD workflows
+
+We should centralize workflows, configurations, and templates as much as possible. Refer to [org-infra](https://github.com/complytime/org-infra).
+
+## 5. Coding Standards
+
+### Guidelines for all programming language
+
+- **Empty Line at End of File**: Ensure that all files include an empty line at the end. This helps with version control diffs and adheres to POSIX standards.
+- The pre-commit and pre-push hooks can be configured by installing [pre-commit](https://pre-commit.com/) and running code repo specific `make` commands.
```

Comment:
Maybe we could split this last part in a single bullet saying that we use `Makefile` to centralize code specific commands?